plugins/versionpress/src/Initialization/WpConfigSplitter.php

```diff
@@ -35,9 +35,10 @@ public static function split($wpConfigPath, $commonConfigName = self::COMMON_CON
         $commonConfigLines = is_file($commonConfigPath)
             ? file($commonConfigPath)
             : file(__DIR__ . '/wp-config.common.tpl.php');
-
+        $constantsClass = new ReflectionClass(PublicConstants::class);
+        $vpPublicConstants = array_values($constantsClass->getConstants());
         // https://regex101.com/r/zD3mJ4/2
-        $constantsForRegex = join('|', self::$constantsForExtraction);
+        $constantsForRegex = join('|', array_merge(self::$wpConstantsForExtraction, $vpPublicConstants));
```

Comment:
We definitely don't want to move all public constants to the common config. For example `VP_ENVIRONMENT` or `VP_GIT_BINARY` are environment-specific. Thus, they need to stay in the `wp-config.php`.